test/interpret.jl

```diff
@@ -1,5 +1,5 @@
 # Contains both positive and negative UTC offsets and observes daylight saving time.
-apia = compile("Pacific/Apia", tzdata["australasia"])
+apia = first(compile("Pacific/Apia", tzdata["australasia"]))
```

Comment:
You had to add `first` a lot. Is there a way to avoid that?

Reply:
I could make:
```julia
function compile_tz(args...)
    tz, class = compile(args...)
    return tz
end
```
I didn't think it was worth it though

Reply:
Yeah probably not I guess